src/state.rs

```diff
@@ -490,7 +490,7 @@ impl DemandProxyState {
     // only support workload
     pub async fn fetch_workload_by_uid(&self, uid: &str) -> Option<Workload> {
         // Wait for it on-demand, *if* needed
-        debug!(%uid, "fetch workload");
+        debug!(%uid, "fetch workload by uid");
```

Comment:
This is to avoid confusion with the `fetch_workload` function. Currently, both functions print the same log.